src/commands/login/create-config.ts

```diff
@@ -83,6 +84,9 @@ export class CreateConfig extends BaseCommand<typeof CreateConfig> {
     }
     // Flag/prompt input
     else {
+      if (flags['no-input']) {
+        throw new CLIError(giveFlagInputErrorMessage('file'))
+      }
```

Comment:
Only the name and redirectUris are required and they could have been provided via flag already. So we should only fail if they have not been provided.

Reply:
Ah! missed this one, nice catch